rosdep/python.yaml

```
@@ -9505,6 +9505,19 @@ python3-typeguard-pip:
   ubuntu:
     pip:
       packages: [typeguard]
+python3-types-pyyaml:
+  debian: [python3-typeshed]
+  fedora:
+    pip:
+      packages: [types-pyyaml]
+  ubuntu: 
+    '*': [python3-typeshed]
+    focal:
+      pip:
+        packages: [types-pyyaml]
+  rhel:
+    pip:
+      packages: [types-pyyaml]
```

via EPEL: https://packages.fedoraproject.org/pkgs/python-types-pyyaml/python3-types-pyyaml/

```suggestion
  rhel:
    '*': [python3-types-pyyaml]
    '8':
      pip:
        packages: [types-pyyaml]
```